apps/dot_ext/tests/test_views.py

```diff
@@ -247,9 +247,11 @@ def test_delete_token_success(self):
             application=application,
         ).exists())
 
-        response = self.client.get('/v1/fhir/Patient',
-                                   HTTP_AUTHORIZATION="Bearer " + tkn.token)
-        self.assertEqual(response.status_code, 403)
+        # Post Django 2.2:  An OSError exception is expected when trying to reach the
+        #                   backend FHIR server and proves authentication worked.
+        with self.assertRaisesRegexp(OSError, "Could not find the TLS certificate file"):
+            response = self.client.get('/v1/fhir/Patient',
+                                       HTTP_AUTHORIZATION="Bearer " + tkn.token)
```

Comment:
After the upgrade, the call to the back end FHIR server attempts a connection and fails. This is the behavior we want, since it proves we passed the authorization stage.